Automattic/jetpack
# Package Unit tests: update test file names to make sure they runs in Travis

Fixes #16350

In package's `phpunit` configuration, we expecting test files with `test` suffix, but for some of these packages, it was not the case. Renaming these test files fixed the issue.

I decided to rename files instead of updating definitions in the sake of consistency.

#### Changes proposed in this Pull Request:

* Rename some test files

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Make sure Travis is green
* Take a look at a job which running package tests. make sure that there is no "No tests executed!" mentions in output

#### Proposed changelog entry for your changes:

* n/a
